### DISCUSSION TITLE
jsTree render two trees on razor view in different divs

### DISCUSSION THREAD

**Discussion by Stefano-Buzzoni** [vakata/jstree#2633] [Q&A]
Hi, I'm using jsTree to render two trees with hierarchical data, data are retrieved by calling two c# methods via ajax using the jsTree instance. My issue is that only the first instance is called whereas the second one is totally ignored and I can't really get a grasp on it.
MVC .NET 6

Edit: sorry for code indentation but this is my first issue on github

`<div id="jobstree" />`
`<div id="costCentersTree" />`

`<script>`

$(function() {

      //called

        $('#jobstree').jstree({
            'core': {
                'data': {
                    'url': function(node) {
                        return "/Home/GetNodes/";
                    },
                    'data': function(node) {
                        return {
                            'id' : node.id,
                            'text': node.text,
                            'children': node.children
                        };
                    }
                },
                "multiple": false,
            },
        });

       //ignored

        $('#costCentersTree').jstree({
            'core': {
                'data': {
                    'url': function(node) {
                        return "/Home/GetCostCenterTree/";
                    },
                    'data': function(node) {
                        return {
                            'id': node.id,
                            'text': node.text,
                            'children': node.children
                        };
                    }
                },
                'multiple': false,
            },
        });
});
`</script>`

**Comment by jplevene** [comment]
What does it say in the browser console?
